lib/templates-sync.php

```diff
@@ -42,6 +47,7 @@ function _gutenberg_create_auto_draft_for_template( $post_type, $slug, $theme, $
 				'post_status'  => 'auto-draft',
 				'post_type'    => $post_type,
 				'post_name'    => $slug,
+				'tax_input'    => array( 'wp_theme' => array( $theme, 'wp_file_based' ) ),
```

Comment:
How about `_wp_file_based` instead?
It should hopefully guard well enough against themes with the same name (and yes, the irony that Automattic has a theme that starts with an underscore is not lost on me 😄).